zomes/rea_agent/lib/src/lib.rs

```diff
@@ -27,9 +29,29 @@ pub fn handle_create_agent<S>(entry_def_id: S, agent: CreateRequest) -> RecordAP
     where S: AsRef<str>
 {
     let (header_addr, base_address, entry_resp): (_,_, EntryData) = create_record(&entry_def_id, agent)?;
+    // TODO: check if contains `link_pub_key` field
+    let pub_key = agent_info()?.agent_latest_pubkey;
+    // TODO: error handling to match expect error type
+    let _pub_key_link = create_link(pub_key, header_addr.clone(), HdkLinkType::Any, ())?;
     construct_response(&base_address, header_addr, &entry_resp, /*get_link_fields(&base_address)?*/)
 }
 
+pub fn handle_get_my_agent() -> RecordAPIResult<ResponseData>
+{
+    let my_pub_key = agent_info()?.agent_latest_pubkey;
```

Comment:
Neat! I didn't expect you to do all of this in the backend, but I'm glad it's happened that way.
Another option would have been to use the generic 'whois' method and pass our own `AgentPubKey` out of the cell and back in again.

Reply:
Can you explain (both to me and Wesley) what the `whois` function is? It is not in the VF Spec I don't think. What purpose does it serve?

Reply:
I am going to move this convo to an issue actually

Reply:
wouldn't we still need some backend solution to resolve an agent pub key to the agent record entry?

Reply:
Not for hREA, but we will eventually need it for non-REA parts of an application which deal with Holochain-native identifiers (read: `AgentPubKey`). So that's what `whois` will be for, and why it's not needed for MMR (: